http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/GoogleComputeDAO.scala

```diff
@@ -23,8 +23,6 @@ trait GoogleComputeDAO {
 
   def getComputeEngineDefaultServiceAccount(googleProject: GoogleProject): Future[Option[WorkbenchEmail]]
 
-  def getGoogleApiServiceAccount(googleProject: GoogleProject): Future[Option[WorkbenchEmail]]
```

Comment:
I figured this didn't really belong in `GoogleComputeDAO` -- I moved this logic to `ClusterHelper`